roles/mantlui/tasks/main.yml

```diff
@@ -30,5 +29,6 @@
     state: started
   notify:
     - restart nginx-mantlui
+    - restart nginx-consul
```

Comment:
without this restart, I see lots of errors in the nginx-consul logs like

```
Jul 18 11:58:28 resching-aws-control-01 docker[18768]: 2016/07/18 11:58:28 [error] 53#0: *2338 connect() failed (111: Connection refused) while connecting to upstream, client: 172.17.0.3, server: , request: "GET /v1/health/service/traefik?index=1347&token=<token>&wait=60000ms HTTP/1.1", upstream: "http://[::1]:8500/v1/health/service/traefik?index=1347&token=<token>&wait=60000ms", host: "consul.service.consul:8500", referrer: "http://consul.service.consul:8500/v1/health/service/traefik?index=1347&token=<token>&wait=60000ms"
```

Restarting nginx-consul resolves the problem but I was unable to figure out why this is necessary. Any ideas?